TSC-Charter.md

```diff
@@ -56,14 +57,29 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC membership shall include (but is not limited to) Maintainers from Core
+Projects as defined in the project lifecycle document and Section 7. The TSC
+may, at its discretion, invite new members to join the TSC who are not
+considered to be Maintainers from Core Projects.
+
+The TSC may, at its discretion, invite any number of non-voting observers to
+participate in TSC meetings.
```

Comment:
Yes there have been cases were observers have been asked to leave for the private section, if this is the expectation then its likely good to document.